insights/client/__init__.py

```diff
@@ -47,17 +56,19 @@ def set_up_logging(self):
     def version(self):
         return "%s-%s" % (package_info["VERSION"], package_info["RELEASE"])
 
+    @_net
     def test_connection(self):
         """
             returns (int): 0 if success 1 if failure
         """
-        return client.test_connection(self.config)
+        return client.test_connection(self.connection)
```

Comment:
This will also break `post_update` function in `v1.py` I believe.

https://github.com/RedHatInsights/insights-core/blob/c1165678a91d58a2fe5d5358fcff3b42453ba1b7/insights/client/phase/v1.py#L148